packages/nx-plugin/src/migrations/update-11-0-0/rename-ng-update-into-nx-migrate.ts

```diff
@@ -0,0 +1,10 @@
+import { Tree, readJson } from '@nrwl/devkit';
+
+export default function update(host: Tree) {
+  const p = readJson(host, 'package.json');
+  if (p['ng-update']) {
+    p['ng-migrate'] = p['ng-update'];
```

Comment:
Hi @vsavkin I know this has already been merged, but I am guessing based on the name of the migration (and other files in this PR) that this was an important typo and it should be:

```
p['nx-migrations'] = p['ng-update'];
```

on line 6?

I think this also highlights that this migration is lacking test coverage